xarray/test/test_quantities.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
```

Comment:
Can you add a docstring explaining that this module uses quantities as a concrete example, but really it's just checking for subclass compatibility.

Also, I might call it something more genericl ike `test_numpy_subclasses.py`.